### PR TITLE
[librealsense]Update the README.md file

### PR DESCRIPTION
**Pull request by minggangw**
To align the format of README.md with other modules' README.

### CODE REVIEW

**Comment by minggangw** [comment]
@kenny-y PTAL, thanks!

File: src/librealsense/README.md [hunk 1]
```diff
@@ -1,46 +1,45 @@
...
+    height: 480,
+    format: 'rgb',
+    framerate: 30,
+  };
+
+  device.on('frameready', (frame) => {
+    // A new frame was captured.
+  });
+
+  device.enableStream('color', mode).then(() => {
```
Comment (kenny-y) [inline src/librealsense/README.md]: Better to have depth stream to directly demonstrate the ability of RealSense camera

File: src/librealsense/README.md [hunk 2]
```diff
@@ -1,46 +1,45 @@
...
+  });
+
+  device.context = context;
+});
+```
 
-Solution: such behavior can be observed sometimes, e.g. a infra-red channel is down and therefore depth channel hangs. You can wait for another while for its self-recovery (yes it sometimes does), or unplug the camera and plug it back again.
+# Document
+Please refer to the [project document home page](https://01org.github.io/node-realsense/), and also the details page listed below:
+- [Concept and terms](https://software.intel.com/sites/products/realsense/person/developer_guide.html#Introduction)
```
Comment (kenny-y) [inline src/librealsense/README.md]: A link to https://github.com/IntelRealSense/librealsense is recommended, PersonTracking link seems no use here...

File: src/librealsense/README.md [hunk 3]
```diff
@@ -1,46 +1,45 @@
...
+    // A new frame was captured.
+  });
+
+  device.enableStream('color', mode).then(() => {
+    device.start().then(() => {
+      console.log('Camera started...');
+    });
+  });
+
+  device.context = context;
```
Comment (kenny-y) [inline src/librealsense/README.md]: I think we should find a good way to workaround the GC issue, and also make the code prettier than this :)

**Review by kenny-y — CHANGES_REQUESTED** [review]

**Review by kenny-y — APPROVED** [review]

**Pull request merged**